.github/workflows/ci.yml

```diff
@@ -33,7 +33,10 @@ jobs:
       with:
         go-version: ${{ matrix.go-version }}
 
-    - run: go test -v ./...
+    - run: go test -coverprofile=coverage.txt -v -race ./...
+
+    - name: Upload coverage reports to Codecov
+      uses: codecov/codecov-action@v3
```

Comment:
```suggestion
    - codecov/codecov-action@v3
```

Reply:
Why should we remove the name? IMO it makes apparent what this step does without actually the reader actually needing to know what Codecov is